### PR TITLE
Finished porting for prefix "t_" files

### PR DESCRIPTION
**Pull request by eddie9712**
* Edited `r_ping.rs` according to some suggestions in #10 PR10
* Replace structopt to clap in prefix "t_" files
* Update the usage of zenoh in prefix "t_" files 
* `Cargo fmt`

- [x] Compilation passed with command cargo build --bin t_ping (t_pong)

- [x]  Verified by the following commands: `./t_pong -e tcp/127.0.0.1:7447 -m peer` ,

`./t_ping -m peer -p 8 -n none -s none -i 1 -e tcp/127.0.0.1:7447`

- [x] Checked with `Cargo clippy`

### CODE REVIEW

**Comment by william-wyliang** [comment]
Also invite @YuanYuYuan to review.

File: latency/src/bin/t_ping.rs [hunk 1]
```diff
@@ -230,13 +256,13 @@ async fn single(opt: Opt, whatami: WhatAmI) {
             reliability: Reliability::Reliable,
         };
         let congestion_control = CongestionControl::Block;
-        let key = ResKey::RName("/test/ping".to_string());
+        let key = "/test/ping".into();
```
Comment (YuanYuYuan) [inline latency/src/bin/t_ping.rs:259]: A plain `into` is meaningless. I suggest you can move to where `key` is used.

**Review by YuanYuYuan — APPROVED** [review]

File: latency/src/bin/t_ping.rs (hunk 1 above)
Comment (eddie9712) [reply latency/src/bin/t_ping.rs:259]: Ok, I will edit it.

**Review by william-wyliang — APPROVED** [review]
No comment other than Yuan's. Please resubmit it and will merge then.

**Comment by eddie9712** [comment]
Edited.

File: latency/src/bin/t_ping.rs [hunk 2]
```diff
@@ -313,20 +326,20 @@ async fn parallel(opt: Opt, whatami: WhatAmI) {
...
+        data.write_all(&count_bytes).unwrap();
+        data.write_all(&payload).unwrap();
         let data: ZBuf = data.into();
         let routing_context = None;
         let reply_context = None;
         let attachment = None;
 
         let message = ZenohMessage::make_data(
-            key,
+            key.into(),
```
Comment (william-wyliang) [inline latency/src/bin/t_ping.rs:345]: I think `KeyExpr` already implemented `fn from(name: &'a str) -> KeyExpr<'a>`. So, the `into()` should be able to perform by the compiler implicitly, right?

Comment (william-wyliang) [reply latency/src/bin/t_ping.rs:345]: Sorry, I was wrong. I mixed this with Deref Coercion. Please ignore this.

**Pull request merged**